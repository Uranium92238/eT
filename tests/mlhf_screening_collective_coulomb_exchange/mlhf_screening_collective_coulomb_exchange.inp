system
   name: H2O string
   charge: 0
   multiplicity: 1
end system

do
   ground state
end do

memory
   available: 8
end memory

solver scf
   algorithm: mo-scf-diis
   energy threshold:   1.0d-10
   gradient threshold: 1.0d-10
   coulomb exchange terms: collective
end solver scf

method
   mlhf
end method

active atoms
   selection type: list
   hf: {1, 2, 3}
end active atoms

geometry
basis: sto-3g
O        1.41500       -2.02200       -0.79800
H        2.29800       -1.66800       -0.90100
H        0.83900       -1.26100       -0.87300
O        5.91500       -2.02200       -0.79800
H        6.79800       -1.66800       -0.90100
H        5.33900       -1.26100       -0.87300
O        10.41500       -2.02200       -0.79800
H        9.53200       -2.37600       -0.69500
H        10.99100       -2.78300       -0.72300
O        14.91500       -2.02200       -0.79800
H        14.03200       -2.37600       -0.69500
H        15.49100       -2.78300       -0.72300
O        19.41500       -2.02200       -0.79800
H        20.29800       -1.66800       -0.90100
H        18.83900       -1.26100       -0.87300
O        23.91500       -2.02200       -0.79800
H        23.03200       -2.37600       -0.69500
H        24.49100       -2.78300       -0.72300
end geometry
