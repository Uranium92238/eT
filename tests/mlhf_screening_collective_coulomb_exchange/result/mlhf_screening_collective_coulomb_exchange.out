


                     eT 1.9 - an electronic structure program

  ------------------------------------------------------------------------
   Author list in alphabetical order:
  ------------------------------------------------------------------------
   J. H. Andersen, A. Balbi, S. Coriani, S. D. Folkestad, T. Giovannini, 
   L. Goletto, T. S. Haugland, A. Hutcheson, I-M. Høyvik, E. F. Kjønstad, 
   H. Koch, R. Matveeva, T. Moitra, R. H. Myhre, A. C. Paul, S. Roet, 
   E. Ronca, M. Scavino, A. K. Schnack-Petersen, A. S. Skeidsvoll, Å. 
   H. Tveten
  ------------------------------------------------------------------------
   J. Chem. Phys. 152, 184103 (2020); https://doi.org/10.1063/5.0004713


   This is eT 1.9.0 Jazzy
  ------------------------------------------------------------
  Configuration date: 2022-10-19 10:25:53 UTC -07:00
  Git branch:         detached HEAD
  Git hash:           57c35dcc31df636729251e2448b7b0d0dd22e1f1
  Fortran compiler:   GNU 11.3.0
  C compiler:         GNU 11.3.0
  C++ compiler:       GNU 11.3.0
  LAPACK type:        SYSTEM_NATIVE
  BLAS type:          SYSTEM_NATIVE
  64-bit integers:    OFF
  OpenMP:             ON
  PCM:                OFF
  Forced batching:    OFF
  Runtime checks:     OFF
  ------------------------------------------------------------


  :: Input file
  ================

     Note: geometry section is excluded from this print

     system
        name: h2o string
        charge: 0
        multiplicity: 1
     end system

     do
        ground state
     end do

     memory
        available: 8
     end memory

     solver scf
        algorithm: mo-scf-diis
        energy threshold:   1.0d-10
        gradient threshold: 1.0d-10
        coulomb exchange terms: collective
     end solver scf

     method
        mlhf
     end method

     active atoms
        selection type: list
        hf: {1, 2, 3}
     end active atoms


  Calculation start:2022-10-19 10:30:33 UTC -07:00

  Running on 2 OMP threads
  Memory available for calculation: 8.000000 GB


  :: MLHF wavefunction
  ====================

  - MLHF settings:

     Occupied orbitals:    Cholesky
     Virtual orbitals:     PAOs

     Cholesky decomposition threshold:  0.10E-01

     MO screening enabled

     ==============================================================================
                                     Geometry (angstrom)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: sto-3g
        1  O     1.415000000000    -2.022000000000    -0.798000000000        1
        2  H     2.298000000000    -1.668000000000    -0.901000000000        2
        3  H     0.839000000000    -1.261000000000    -0.873000000000        3
        4  O     5.915000000000    -2.022000000000    -0.798000000000        4
        5  H     6.798000000000    -1.668000000000    -0.901000000000        5
        6  H     5.339000000000    -1.261000000000    -0.873000000000        6
        7  O    10.415000000000    -2.022000000000    -0.798000000000        7
        8  H     9.532000000000    -2.376000000000    -0.695000000000        8
        9  H    10.991000000000    -2.783000000000    -0.723000000000        9
       10  O    14.915000000000    -2.022000000000    -0.798000000000       10
       11  H    14.032000000000    -2.376000000000    -0.695000000000       11
       12  H    15.491000000000    -2.783000000000    -0.723000000000       12
       13  O    19.415000000000    -2.022000000000    -0.798000000000       13
       14  H    20.298000000000    -1.668000000000    -0.901000000000       14
       15  H    18.839000000000    -1.261000000000    -0.873000000000       15
       16  O    23.915000000000    -2.022000000000    -0.798000000000       16
       17  H    23.032000000000    -2.376000000000    -0.695000000000       17
       18  H    24.491000000000    -2.783000000000    -0.723000000000       18
     ==============================================================================

     ==============================================================================
                                     Geometry (a.u.)
     ==============================================================================
        Atom           X                  Y                  Z         # in input
     ==============================================================================
        Basis: sto-3g
        1  O     2.673962466260    -3.821026223871    -1.508001447403        1
        2  H     4.342590634251    -3.152063175775    -1.702643238233        2
        3  H     1.585480218510    -2.382944643077    -1.649730906745        3
        4  O    11.177730026802    -3.821026223871    -1.508001447403        4
        5  H    12.846358194793    -3.152063175775    -1.702643238233        5
        6  H    10.089247779053    -2.382944643077    -1.649730906745        6
        7  O    19.681497587345    -3.821026223871    -1.508001447403        7
        8  H    18.012869419354    -4.489989271967    -1.313359656573        8
        9  H    20.769979835095    -5.259107804665    -1.366271988061        9
       10  O    28.185265147888    -3.821026223871    -1.508001447403       10
       11  H    26.516636979897    -4.489989271967    -1.313359656573       11
       12  H    29.273747395637    -5.259107804665    -1.366271988061       12
       13  O    36.689032708431    -3.821026223871    -1.508001447403       13
       14  H    38.357660876422    -3.152063175775    -1.702643238233       14
       15  H    35.600550460681    -2.382944643077    -1.649730906745       15
       16  O    45.192800268973    -3.821026223871    -1.508001447403       16
       17  H    43.524172100983    -4.489989271967    -1.313359656573       17
       18  H    46.281282516723    -5.259107804665    -1.366271988061       18
     ==============================================================================

  - Cholesky decomposition of AO overlap to get linearly independent AOs:

     Linear dependence threshold:             0.10E-05
     Number of atomic orbitals:               42
     Number of orthonormal atomic orbitals:   42

  - Molecular orbital details:

     Number of occupied orbitals:        30
     Number of virtual orbitals:         12
     Number of molecular orbitals:       42


  Generating initial SAD density
  ==============================


  Determining reference state
  ===========================

  - Setting initial AO density to sad

     Energy of initial guess:              -448.107022012912
     Number of electrons in guess:           60.000000000000

  - Active orbital space:

      Number of active occupied orbitals:        5
      Number of active virtual orbitals:         3
      Number of active orbitals:                 8

  - Screening and integral thresholds:

     Coulomb screening threshold:    0.1000E-15
     Exchange screening threshold:   0.1000E-13
     ERI cutoff:                     0.1000E-15
     One-electron integral  cutoff:  0.1000E-20
     Cumulative Fock threshold:      0.1000E+01

  - SCF solver settings:

     Maximum iterations:                   100
     Acceleration type:                   diis

  - Convergence thresholds

     Residual threshold:            0.1000E-09
     Energy threshold:              0.1000E-09

  - DIIS tool settings:

     DIIS dimension:   8

     Storage (solver scf_errors): memory
     Storage (solver scf_parameters): memory

  Iteration       Energy (a.u.)      Max(grad.)    Delta E (a.u.)
  ---------------------------------------------------------------
     1          -449.546150858738     0.5400E+00     0.4495E+03
     2          -449.584876546543     0.5395E-01     0.3873E-01
     3          -449.585298685713     0.1238E-01     0.4221E-03
     4          -449.585321962209     0.1714E-03     0.2328E-04
     5          -449.585321970899     0.5908E-04     0.8690E-08
     6          -449.585321972542     0.2448E-05     0.1643E-08
     7          -449.585321972545     0.6539E-07     0.3126E-11
     8          -449.585321972545     0.1485E-07     0.0000E+00
     9          -449.585321972545     0.1707E-08     0.5684E-13
    10          -449.585321972545     0.5353E-09     0.0000E+00
    11          -449.585321972545     0.1137E-09     0.5684E-13
    12          -449.585321972545     0.1028E-10     0.5684E-13
  ---------------------------------------------------------------
  Convergence criterion met in 12 iterations!

  - Summary of MLHF wavefunction energetics (a.u.):

     HOMO-LUMO gap:                  0.992843220774
     Nuclear repulsion energy:     158.230432129450
     Electronic energy:           -607.815754101996
     Total energy:                -449.585321972545

  - Summary of MLHF active/inactive contributions to electronic energy (a.u.):

     Active energy:               -111.129949238180
     Active-inactive energy:        26.974330971521
     Inactive energy:             -523.660135835336

  Peak memory usage during the execution of eT: 238.868 KB

  Total wall time in eT (sec):              0.17400
  Total cpu time in eT (sec):               0.19938

  Calculation end:2022-10-19 10:30:33 UTC -07:00

  - Implementation references:

     eT: https://doi.org/10.1063/5.0004713

  eT terminated successfully!
